api/v1alpha1/scheduledscaling_types.go

```diff
@@ -0,0 +1,166 @@
+/*
+MIT License
+
+Copyright (c) 2023 mercari
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+
+*/
+
+package v1alpha1
+
+import (
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
+// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
+
+// ScheduledScalingSpec defines the desired state of ScheduledScaling
+type ScheduledScalingSpec struct {
+	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
+	// Important: Run "make" to regenerate code after modifying this file
+
+	/// The schedule in Cron format, see https://en.wikipedia.org/wiki/Cron.
+	Schedule Schedule `json:"schedule" protobuf:"bytes,1,name=schedule"`
+
+	//TargetRef is the targets that need to be scheduled
+	TargetRefs TargetRefs `json:"targetRefs" protobuf:"bytes,2,name=targetRefs"`
+
+	//Currently only static, might implement others(?)
+	Strategy Strategy `json:"strategy" protobuf:"bytes,3,name=strategy"`
+}
+
+// ScheduledScalingStatus defines the observed state of ScheduledScaling
+type ScheduledScalingStatus struct {
+	// INSERT ADDITIONAL STATUS FIELD - define observed state of cluster
+	// Important: Run "make" to regenerate code after modifying this file
+
+	//Target tortoise phase
+	TortoisePhase TortoisePhase `json:"tortoisePhase" protobuf:"bytes,1,name=tortoisePhase"`
```

Comment:
Actually do we need TortoisePhase here? I'm not sure of this one too, maybe we can remove it as well until we need.